include/seastar/util/defer.hh

```diff
@@ -43,7 +43,7 @@ namespace seastar {
 
 template <typename Func>
 SEASTAR_CONCEPT( requires deferrable_action<Func> )
-class deferred_action {
+class [[nodiscard("unassigned deferred_action")]] deferred_action {
```

Comment:
There's also `deferred_close` and `deferred_<something_else>` ... checking ... `deferred_stop` :)

Reply:
@xemul all added!